swagger.yaml

```diff
@@ -645,20 +645,27 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
-  "/severa/users/{severaUserId}/flextime":
+  "/users/{userId}/flextime":
```

Comment:
This is good, just remember you will need to update the path in the get-flextime-by-user/index.ts in the lambdas